tools/depends/target/curl/Makefile

```diff
@@ -22,6 +22,7 @@ CONFIGURE=cp -f $(CONFIG_SUB) $(CONFIG_GUESS) .; \
                       --without-gsasl \
                       --without-hyper \
                       --without-ngtcp2 \
+                      --without-nghttp2 \
                       --without-nghttp3 \
```

Comment:
You don't need to disable nghttp2 with the patch

Reply:
Oh, I see. Should I just git reset and force push it out then? :)

Reply:
Yeah,